src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -2791,18 +2803,18 @@ subroutine PhotosynthesisHydraulicStress ( bounds, fn, filterp, &
       kcha    = 79430._r8
       koha    = 36380._r8
       cpha    = 37830._r8
-      vcmaxha = 72000._r8
-      jmaxha  = 50000._r8
-      tpuha   = 72000._r8
+      vcmaxha = 73637._r8
+      jmaxha  = 50300._r8
+      tpuha   = 73637._r8
       lmrha   = 46390._r8
 
       ! High temperature deactivation, from:
       ! Leuning (2002) Plant, Cell and Environment 25:1205-1210
       ! The factor "c" scales the deactivation to a value of 1.0 at 25C
 
-      vcmaxhd = 200000._r8
-      jmaxhd  = 200000._r8
-      tpuhd   = 200000._r8
+      vcmaxhd = 149252._r8
+      jmaxhd  = 152044._r8
+      tpuhd   = 149252._r8
```

Comment:
Should the above comment be updated to reflect the appropriate source of these updated constants?